lib/zappifest.rb

```diff
@@ -87,16 +90,28 @@
   c.action do |args, options|
     options.default access_token: ENV["ZAPP_TOKEN"]
     options.default manifest: args.first
-
+    
     VersionHelper.new(c).check_version
     NetworkHelpers.validate_token(options.access_token) unless options.override_url
+    current_user = NetworkHelpers.current_user(options.access_token).body
```

Comment:
Now I know why it fails:
1. Before we had `NetworkHelpers.validate_token(options.access_token) unless options.override_url`
and `current_user` was _never_ called locally because of the `unless`
2. Now when we get to `current_user` it fails since it tries to access
`#{ACCOUNTS_URL}/api/v1/users/current.json` 
3. ACCOUNTS_URL = "https://accounts.applicaster.com/api/v1" 😢 
4. We should change the ACCOUNTS_URL to use `accounts.test` locally and then `current_user` should be okay

Bottom line:
- Local publishing NEVER used the local zapp token in `accounts.test`. I think it should (in a totally different PR)

Reply:
@simonvaucher add at your local envs `ACCOUNTS_URL` to accounts.test

Reply:
How will that help? Line 6 in NetworkHelpers is:
`ACCOUNTS_URL = "https://accounts.applicaster.com/api/v1"`

Reply:
We can always _fix_ this line to this:
```
ACCOUNTS_URL = ENV["ACCOUNTS_URL"] || ""https://accounts.applicaster.com/api/v1"
```
+ modify the README

Reply:
Fixed in the meanwhile with #162